client/layouts/auth.vue

```diff
@@ -25,7 +25,7 @@ useMutationObserver(child, (mutations) => {
         {{ t('app_name') }}
       </h1>
 
-      <MorphAnimatedContainer :prevent-transition="!cardTransition">
+      <MorphAnimatedContainer :prevent-transition="!cardTransition" transition-duration="0.5s">
```

Comment:
.5s ? not .4s. 

What I actually don't understand is how SlideTransitionContainer has a FIXED transition of .4s, and how the MorphAnimatedContainer has default manual transition of .4s too. Are they related ? If so, why is one manual and the other is not. Wouldn't that cause desyncs ?

Reply:
Well, morph and slide are .4s by default, but on the auth pages I extended the transition to .5s to make it more smooth and sweeping. In the auth layout, I fixed the morph duration using attribute and the slide is a dedicated CSS transition with .5s duration.

Reply:
oke